app/src/main/res/layout/fragment_homepage.xml

```diff
@@ -73,87 +74,85 @@
                 android:layout_width="233dp"
-                android:layout_height="91dp"
+                android:layout_height="83dp"
                 android:gravity="left"
-                android:text="When the switch is positioned to the left and lacks a purple color, the switch indicates the statement is false. "
+                android:text="When the switch is positioned to the left and lacks a purple color, the switch tells us that the statement is false. "
                 android:textColor="@color/white"
-                android:textSize="17sp"
-                android:translationX="10dp"
-                android:translationY="-150dp"
+                android:textSize="15sp"
+                android:translationX="150dp"
+                android:translationY="-140dp"
                 tools:ignore="HardcodedText,RtlHardcoded" />
 
             <TextView
                 android:id="@+id/Switch_on_info"
                 android:layout_width="244dp"
-                android:layout_height="82dp"
+                android:layout_height="wrap_content"
                 android:gravity="left"
-                android:text="When the switch is positioned to the right with a purple color, it signifies that the statement is true."
+                android:text="When the switch is positioned to the right with a purple color, it tells us that that the statement is true."
                 android:textColor="@color/white"
-                android:textSize="17sp"
+                android:textSize="15sp"
                 android:translationX="150dp"
-                android:translationY="-140dp"
+                android:translationY="-110dp"
                 tools:ignore="HardcodedText,RtlHardcoded" />
 
             <ImageView
                 android:id="@+id/Switch_on"
                 android:layout_width="125dp"
                 android:layout_height="wrap_content"
                 android:src="@drawable/switchon"
-                android:translationX="10dp"
-                android:translationY="-220dp" />
+                android:translationX="0dp"
+                android:translationY="-180dp" />
 
             <ImageView
                 android:id="@+id/Plus"
                 android:layout_width="70dp"
                 android:layout_height="70dp"
                 android:src="@drawable/plus"
-                android:translationX="270dp"
-                android:translationY="-210dp" />
+                android:translationX="25dp"
+                android:translationY="-160dp"  />
 
             <TextView
                 android:id="@+id/Plus_info"
-                android:layout_width="248dp"
+                android:layout_width="204dp"
                 android:layout_height="63dp"
                 android:gravity="left"
-                android:text="Use the plus icon to indicate the ascending numerical value related to the statement."
+                android:text="Use the plus icon to show the increase of that statement."
```

Comment:
statement is still a slightly unclear depiction of the plus button, maybe try changing it to value